src/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,38 @@
+import { ChangeEvent } from 'react';
+
+import style from './FormInput.module.scss';
+
+type FormInputProps = {
+  type: string;
+  name: string;
+  placeholder: string;
+  value: string;
+  error?: boolean;
+  errorMessage?: string;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
You made value and onChange required. Please, read about controlled vs uncontrolled forms/form elements